docs/guide/contributing.rst

```diff
@@ -192,7 +187,7 @@ The Write the Docs community is available for help, questions, or discussion:
 .. _Fork the Write the Docs www project: https://github.com/writethedocs/www/fork
 .. _file a GitHub issue: https://github.com/writethedocs/www/issues/new
 .. _Submit an issue: https://github.com/writethedocs/www/issues/new
-.. _reStructuredText (.rst) Markup: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
+.. _reStructuredText (.rst): https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
```

Comment:
Just noticed, is there a reason the Markdown link goes to the WTD intro but the rST one doesn't? There is a [WTD intro to rST](https://writethedocs-www--2173.org.readthedocs.build/guide/writing/reStructuredText/)

Reply:
I'm not sure. I thought I used whatever links were already present in the document. I'll review to see which feels like a better source to link to and adjust accordingly in my next commit. Thanks for bringing this up.

Reply:
Updated in latest commit.